src/components/switch/readme.md

```diff
@@ -0,0 +1,75 @@
+# gx-switch
+
+An input element for switch/toggle options.
```

Comment:
I'd change this for something like:

A toggle switch control that enables to select between two options.